content/2022/prepoznavanje-znakovnog-jezika.md

```diff
@@ -2,3 +2,46 @@
 title: Prepoznavanje znakovnog jezika
 summary: Prepoznavanje znakovnog jezika je projekat rađen na letnjem kampu za stare polaznike 2022. godine od Zlate Stefanović i Vladana Bašića.
 ---
+
+### Uvod
+
+Tip neverbalne komunikacije najčešće korišćen među gluvo-nemim ljudima se zove znakovni jezik. Ovaj vid komunikacije omogućava prenos informacija bez korišćenja reči, samo pomoću pokreta i gestikulacija. Međutim, znakovni jezik nije poznat široj populaciji što otežava njegovo razumevanje. Da bi se prevazišao ovaj problem, predlažemo korišćenje savremene tehnologije.
+
+Značajan broj istraživanja je urađeno na ovu temu - rad [1] se bavio povećavanjem dubina ConvNet-a sa jako malim (3x3) konvolucionim filterima. Pri ovome dobijen je 6.7% validation error. U [2] je razmatran Indijski Znakovni Jezik (ISL). Urađena je detekcija i praćenje šake na osnovu boje kože, potom je korišćen Grid za ekstraktovanje obeležja na kraju čeka su gestikulacije klasifikovane kNN algoritmom. Ovako je dobijena tačnost od 99.7%. Rad [3] bio je fokus na detektovanju položaja u kojima nije pokazan nijedan znak koristeći Novelty Detection, pri čemu je dobijena tačnost od 97.59%.
+
+### Baza podataka
+
+Za bazu podataka korišćena je baza sintetički generisanih slika američkog znakovnog jezika [[4]](kaggle.com/datasets/lexset/synthetic-asl-alphabet). Korišćena je sintetička baza zbog velikog broja slika različitih pozadina , osvetljenja i boja kože u nadi da će modeli, kao posledica veće raznovrsnosti, biti više robustni. Baza se sastoji od 27000 slika dimenzija 512 x 512 piksela podeljena na trening i test setove. Podeljena je na 27 foldera koji predstavljaju 27 klasa. Svako folder sadrži 900 trening i 100 test primera.
+
+### kNN - k Nearest Neighboors
+
+##### Obrada baze
+
+Da bi se detektovao ceo region slike na kome se nalazi šaka, a samim tim i znak šake klasifikovao, tražena je boja kože. Vrednost ove boje će biti predstavljena kao opseg - različit je za svaku sliku jer se u bazi mogu pronaći slike sa senkama i slike šaka drugačijih tonova kože. Za olakšanje ovog procesa ekstraktovane su 21 ključne tačke šake (ukupno 42 koordinate) pomoću MediaPipe Holistic Pipeline-a. Prvi pristup za utvrđivanje boje kože je uzimanje srednje vrednosti dobijenih 42 tačaka, pri čemu su dobijeni neprecizni rezultati. Drugi način bio je odredjivanje koordinate sredine šake i uzimanje njene vrednosti, što nije radilo jer se često nalazila senka na tom delu slike. Finalni i najprecizniji način je bio uzimanje celog opsega ovih tačaka.
```

Comment:
Prva rečenica u ovom pasusu zvuči kao da je činjenica da ako želimo da detektujemo region šake moramo da koristimo baš vašu metodu. Bolje da počnete sa "U ovom radu smo detektovali region slike na kome se nalazi šaka tom i tom metodom...".

Dodajte referencu na MediaPipe Holistic Pipeline jer nije nešto što se podrazumeva da ljudi znaju.

Ako tvrdite da su dobijeni neprecizni rezultati ili da je detektovana senka dostavite i neki dokaz. Ili to uradite ili samo navedite šta ste na kraju koristili za utvrđivanje boje kože

Reply:
prva dva su vec popravljena, za treci smo naveli da smo ovaj opseg koristili, jel to okej ili treba jos nesto da se doda

Reply:
Ok je, možda još da dodate koji ste tačno opseg koristili